### ISSUE TITLE
Decide on uniform units of data for end user

### ISSUE THREAD

**Issue by easherma** [chihacknight/How-Renewable-Is-My-State#1]
We've narrowed down our data points, but some of them appear to be in varying units. A consistent unit of measure will be easier for users to understand.

**Comment by brendanscully64** [comment]
Agreed. This should already be the case -- all MSN codes ending in "B" are in billions of BTU.
